modules/ROOT/pages/_partials/commons/common-vector-search.adoc

```diff
@@ -0,0 +1,217 @@
+You can use Vector Indexes to efficiently retrieve vectors, similar to a target vector.
+Before use, a Vector Index needs to be trained to compute the centroids and parameters for encoding the vectors.
+
+NOTE: If there are insufficient vectors in the database for training, a warning message will be logged, indicating the required number of vectors.
+
+Training is configured to commence automatically on the first Vector Search query when there are enough vectors to be trained based on the `minimum-training-size` configuration.
+Be aware that vector index training can affect query performance.
+If a query is executed against the index before it is trained, a full scan of the vectors will be performed. 
+
+
+
+=== About Lazy Vector Indexes
+
+IMPORTANT: Lazy index is not an automatic process, you will need to manually schedule the index updates. 
+
+Lazy vector indexes (lazy index) is Couchbase Lite specific functionality that updates indexes asynchronously, satisfying the following use cases:
+
+* Documents have been added to the application by the end-user with no available Machine Learning (ML) model to generate the vectors.
+You can use lazy indexing to schedule updates to the index of said documents when a ML model is available. 
+
+* The remote ML model used stops working or has intermittent availability, causing a failed update.
+With lazy index, you can skip documents that fail to update and schedule the index process to be retried at a later date.
+
+Lazy indexing is an asynchronous process that provides developers full control of:
+
+* When to update the index.
+
+* The number of vectors to update to the index.
+
+* Whether to cancel or skip certain indexes when the model is unavailable or has failed.
+
+NOTE: Updating in lazy index is an independent process from saving document operations.
+
+See here for examples of how to xref:{param-module}:working-with-vector-search.adoc#create-a-lazy-vector-index[use lazy index in your applications].
+
+Lazy indexing is an alternate approach to using the standard predictive model with regular vector indexes which handle the indexing process automatically. 
+The table below compares the two processes.
+
+.Regular Vector Indexes vs Lazy Vector Indexes
+[cols ="3*"]
+|===
+|Feature |Regular Index |Lazy Index
+
+|Update when documents are changed
+|image:ROOT:yes.png[]
+|image:ROOT:no.png[]
+
+|Update when documents are deleted or purged
+|image:ROOT:yes.png[]
+|image:ROOT:yes.png[]
+
+|The application has control when to update the index
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|The application can skip updating the index
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|Is an asynchronous process
+|image:ROOT:no.png[]
+|image:ROOT:yes.png[]
+
+|===
+
+=== About Vector Encoding
+
+Vector encoding reduces the size of Vectors index by algorithmic compression.
+You can configure the Vector Encoding in Couchbase Lite depending on your application's needs.
```

Comment:
Change "depending on your" to "to address your"

Reply:
Done